### PR TITLE
add a readme with some explanation.

### PR DESCRIPTION
**Pull request by fiatjaf**
I am not sure I got it right, let me know!

Formatted README: https://github.com/fiatjaf/sbt-npm-dependencies/blob/readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +8,46 @@ To use sbt-npm-dependencies in an existing SBT project with Scala 2.12 or a late
...
+  "secondNpmPackageName" -> "version",
+  ...
+)
+```
+
+## Internals
+
+(For integration with other build systems)
+
+sbt-npm-dependencies will export a file called `NPM_DEPENDENCIES` containing the dependencies in the following format:
```
Comment (armanbilge) [inline README.md:26]: Where is this file exported to?

File: README.md [hunk 2]
```diff
@@ -10,3 +8,46 @@ To use sbt-npm-dependencies in an existing SBT project with Scala 2.12 or a late
...
+    { "secondNpmPackageName": "version" },
+    ...
+  ],
+  "test-devDependencies": [
+    { "npmPackageName": "version" },
+    { "secondNpmPackageName": "version" },
+    ...
+  ]
+}
+```
```
Comment (armanbilge) [inline README.md:51]: lol, I think the code is bugged
https://github.com/davenverse/sbt-npm-dependencies/blob/81d0def60664341d9b8a7f9e29a5369eeeacbb28/core/src/main/scala/io/chrisdavenport/sbt/npmdependencies/NpmDependencies.scala#L54-L61

File: README.md [hunk 3]
```diff
@@ -10,3 +8,46 @@ To use sbt-npm-dependencies in an existing SBT project with Scala 2.12 or a late
...
+)
+```
+
+## Internals
+
+(For integration with other build systems)
+
+sbt-npm-dependencies will export a file called `NPM_DEPENDENCIES` containing the dependencies in the following format:
+
+```js
```
Comment (armanbilge) [inline README.md:28]:
```suggestion
```json
```

Comment (fiatjaf) [reply README.md:28]: I used `js` so it doesn't scream on the `...`.

File: README.md (hunk 1 above)
Comment (fiatjaf) [reply README.md:26]: To the jar top-level. I don't know how to say this.

Comment (armanbilge) [reply README.md:26]: "to the jar" is fine

**Pull request merged**